pkg/cloud/gcp/actuators/machine/machine_scope_test.go

```diff
@@ -0,0 +1,445 @@
+package machine
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-api-provider-gcp/pkg/apis/gcpprovider/v1beta1"
```

Comment:
Nit: Would be better to alias this when we import so that we don't have a random `v1beta1` amongst all the other APIs, `gcpv1` might be a better alias. We've also imported it elsewhere as `gcp`